camelot/model/party.py

```diff
@@ -211,11 +211,12 @@ class City( GeographicBoundary ):
     """A subclass of GeographicBoundary used to store the name, the postal code
     and the Country of a city"""
     __tablename__ = 'geographic_boundary_city'
-    country = ManyToOne( Country, required = True, ondelete = 'cascade', onupdate = 'cascade' )
-    geographicboundary_id = Field( camelot.types.PrimaryKey(),
-                                   ForeignKey('geographic_boundary.id'),
-                                   primary_key = True,
-                                   autoincrement = False )
+    country_geographicboundary_id = schema.Column(sqlalchemy.types.Integer(),
+                                                  schema.ForeignKey(Country.geographicboundary_id, ondelete='cascade', onupdate='cascade'),
+                                                  nullable=False, index=True)
+    country = orm.relationship(Country, backref='city', foreign_keys=[country_geographicboundary_id])
+    geographicboundary_id = schema.Column(sqlalchemy.types.Integer(),schema.ForeignKey(GeographicBoundary.id),
+                                          primary_key=True, nullable=False)
```

Comment:
is de autoincrement=False hier nt vergeten ?

Reply:
Ik dacht dat Pieter-Jan had gezegd dat die weg mocht (maar dat kan ook bij iets anders zijn geweest)